Manifest namespace

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':metamap_plugin_flutter'.
> Could not create an instance of type com.android.build.api.variant.impl.LibraryVariantBuilderImpl.
   > Namespace not specified. Specify a namespace in the module's build file. See https://d.android.com/r/tools/upgrade-assistant/set-namespace for information about setting the namespace.
     
     If you've specified the package attribute in the source AndroidManifest.xml, you can use the AGP Upgrade Assistant to migrate to the namespace value in the build file. Refer to https://d.android.com/r/tools/upgrade-assistant/agp-upgrade-assistant for general information about using the AGP Upgrade Assistant.

* Try:
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -24,6 +24,7 @@ apply plugin: 'kotlin-android'
 
 android {
     compileSdkVersion 31
+    namespace "com.getmati.metamap_plugin_flutte"
 
     sourceSets {
         main.java.srcDirs += 'src/main/kotlin'

--- a/android/src/main/AndroidManifest.xml
+++ b/android/src/main/AndroidManifest.xml
@@ -1,3 +1,2 @@
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-  package="com.getmati.metamap_plugin_flutter">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
 </manifest>

--- a/pubspec.yaml
+++ b/pubspec.yaml
@@ -1,7 +1,7 @@
 name: metamap_plugin_flutter
 description: Flutter plugin for Mati SDK.
 
-version: 4.3.2
+version: 4.3.3
 homepage: 'https://github.com/GetMati/mati-flutter-plugin'
 
 environment:

